src/features/commands.ts

```diff
@@ -638,47 +638,119 @@ Instead:
+              color: EMBED_COLOR,
+            },
+          ],
+        })
+
+      } else {
+        // permission overwrites can only be applied on Guild Text Channels
+        if (msg.channel.type === "GUILD_TEXT") {
+          const { channel: guildTextChannel } = msg;
+          await guildTextChannel.permissionOverwrites.create(
+            guildTextChannel.guild.roles.everyone,
+            {
+              SEND_MESSAGES: false,
+              CREATE_PUBLIC_THREADS: false,
+              CREATE_PRIVATE_THREADS: false,
+              SEND_MESSAGES_IN_THREADS: false,
+            },
+          );
+          guildTextChannel.send("This channel has been locked by a moderator");
+        }
       }
     },
   },
   {
     words: ["!unlock"],
-    help: "",
-    category: "Communication",
+    help: "used for unlocking channel",
+    category: "Moderation",
     handleMessage: async (msg) => {
       if (!msg.guild || !isStaff(msg.member)) {
-        return;
+        await msg.react("💩")
+        await msg.channel.send({
+          embeds: [
+            {
+              title: "tsk tsk",
+              type: "rich",
+              description: `${msg.author.toString()} This command is reserved for staff, Dont try to do this kind of shit`,
+              color: EMBED_COLOR,
+            },
+          ],
+        })
+      } else {
+        // permission overwrites can only be applied on Guild Text Channels
+        if (msg.channel.type === "GUILD_TEXT") {
+          const { channel: guildTextChannel } = msg;
+          await guildTextChannel.permissionOverwrites.create(
+            guildTextChannel.guild.roles.everyone,
+            {
+              SEND_MESSAGES: null, // null will inherit the permission from the parent channel category
+            },
+          );
+        }
       }
-
-      // permission overwrites can only be applied on Guild Text Channels
-      if (msg.channel.type === "GUILD_TEXT") {
-        const { channel: guildTextChannel } = msg;
-        guildTextChannel.permissionOverwrites.create(
-          guildTextChannel.guild.roles.everyone,
-          {
-            SEND_MESSAGES: null, // null will inherit the permission from the parent channel category
-          },
-        );
+    },
+  },
+  {
+    words: ["!ban"],
+    help: "used for banning users \n\t(usage: `!ban reason user`)",
+    category: "Moderation",
+    handleMessage: (msg) => {
+      if (!msg.guild || !isStaff(msg.member)) {
+        msg.react("💩")
+        msg.channel.send({
+          embeds: [
+            {
+              title: "tsk tsk",
+              type: "rich",
+              description: `${msg.author.toString()} This command is reserved for staff, Dont try to do this kind of shit`,
+              color: EMBED_COLOR,
+            },
+          ],
+        })
+      } else {
+        const [command, reason, userid] = msg.content.split(" ")
```

Comment:
`command` needs to be omitted since it's not used

Reply:
Ya it's not in use but in array the command also comes. So, fro work all things i need to assign it some where

Reply:
Ok i will try to remove it

Reply:
done !